src/main/java/productDependency/CreateDependentCategoryPage.java

```diff
@@ -0,0 +1,974 @@
+		private WebElement enterMin3;
+
+		/**
+		 * Method to Enter Price Rate.
+		 * 
+		 * @throws IOException
+		 */
+		public void enterMin3() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter Min value");
+			JavaScriptExec.sleep();
+			WaitClass.WaitForElementisDisplay(driver, 10, enterMin3);
+			Assert.assertTrue(enterPriceRate.isDisplayed());
+			enterMin3.clear();
+			enterMin3.sendKeys("2");
+		}
+		
+		@FindBy(how = How.XPATH, using = "//*[@id='product.dependencyMin']")
+		private WebElement entermin6;
+
+		
+		public void entermin6() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter Min value");
+			JavaScriptExec.sleep();
+			WaitClass.WaitForElementisDisplay(driver, 10, entermin6);
+			Assert.assertTrue(enterPriceRate.isDisplayed());
+			entermin6.clear();
+			entermin6.sendKeys("0");
+		}
+		
+		private WebElement selectProductDependency2;
+
+		/**
+		 * Method to select product in dependencies section.
+		 * 
+		 * @throws IOException
+		 */
+		public void selectProductDependency2() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			JavaScriptExec.sleep();
+			driver.findElement(By.xpath("//*[@id='product.dependencyItems']/option[3]")).click();
+		}
+		
+		@FindBy(how = How.XPATH, using = "//*[@id='product.dependencyMax']")
+		private WebElement enterMax3;
+
+		/**
+		 * Method to Enter Price Rate.
+		 * 
+		 * @throws IOException
+		 */
+		public void enterMax3() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter Max value");
+			JavaScriptExec.sleep();
+			WaitClass.WaitForElementisDisplay(driver, 10, enterMax3);
+			Assert.assertTrue(enterPriceRate.isDisplayed());
+			enterMax3.clear();
+			enterMax3.sendKeys("3");
+		}
+		
+		@FindBy(how = How.XPATH, using = "//*[@id='product.dependencyMax']")
+		private WebElement enterMax4;
+
+		/**
+		 * Method to Enter Price Rate.
+		 * 
+		 * @throws IOException
+		 */
+		public void enterMax4() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter Max value");
+			JavaScriptExec.sleep();
+			WaitClass.WaitForElementisDisplay(driver, 10, enterMax4);
+			Assert.assertTrue(enterPriceRate.isDisplayed());
+			enterMax4.clear();
+			enterMax4.sendKeys("5");
+		}
+		
+		@FindBy(how = How.XPATH, using = "//*[@id='product.dependencyMax']")
+		private WebElement enterMax6;
+
+		/**
+		 * Method to Enter Price Rate.
+		 * 
+		 * @throws IOException
+		 */
+		public void enterMax6() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter Max value");
+			JavaScriptExec.sleep();
+			WaitClass.WaitForElementisDisplay(driver, 10, enterMax6);
+			Assert.assertTrue(enterPriceRate.isDisplayed());
```

Comment:
@mayankchhajer There is alot of redundancy in the code, Please reduce the code when you have to use same xpath again and again. Don't repeat same xpath's method again